guides/common/modules/proc_adding-upstream-repositories-for-deb.adoc

```diff
@@ -0,0 +1,44 @@
+[id="adding-upstream-repositories-for-{DL}-{DL-major-context}_{context}"]
+= Adding upstream repositories for {DL} {DL-major}
+
+This example creates a product and repositories for {DL} {DL-major}.
+
+.Prerequisites
+* xref:Extracting_GPG_Public_Key_Fingerprints_from_Release_Files_{context}[Extract the GPG public keys] from the `Release` file
+* xref:Importing_a_Custom_GPG_Key_{context}[Import the content credentials] into {Project}
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click *Create Product* to create a product named `{DL} {DL-major}`.
+For more information, see xref:Creating_a_Custom_Product_{context}[].
+. On the *Repositories* tab, click *New Repository* to create three repositories of type `deb` as follows:
```

Comment:
This is how I would have written it. But you can ignore my suggestion because the rest of the procedures are written exactly the same way. :-)

```suggestion
. On the *Repositories* tab, click *New Repository* to create three `deb` repositories with the following parameter values:
```

Reply:
Applied. But I kinda dislike "click X to create three repositories" when it's actually "click X to create one repo; redo twice". What do you think?